app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -198,7 +200,7 @@ ledgerRecoveryFailedTitle=Recovery Failed
 ledgerRecoveryFailedMessage=Please re-enter keys or try different keys.
 ledgerRecoveryNetworkFailedTitle=Network Error
 ledgerRecoveryNetworkFailedMessage=Please check your internet connection and try again.
-ledgerRecoveryContent=The balance of the recovered wallet will be transferred to your new Brave wallet. The old wallet will still exist as an empty wallet.
+ledgerRecoveryContent=Your previous wallet will now be used. Your new wallet will be discarded.
```

Comment:
correct. wallet recovery is now actually "recovery of old wallet" rather than "transfer the balance of the old wallet to the new wallet"